types/styled-jsx/tslint.json

```diff
@@ -0,0 +1,7 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
+        "dt-header": false,
```

Comment:
For new packages, this file should only `"extends": "dtslint/dt.json"`

Reply:
@plantain-00 fixed